homework-functions/index.js

```diff
@@ -0,0 +1,93 @@
+class TrieNode {
+  constructor(key) {
+    this.key = key;
+    this.parent = null;
+    this.children = {};
+    this.end = false;
+    this.count = null;
+  }
+  getWord() {
```

Comment:
Looks like `TrieNode` should be as simple as possible and shouldn't know about words and etc